examples/nats-bench/main.go

```diff
@@ -34,7 +36,12 @@ const (
 )
 
 func usage() {
-	log.Fatalf("Usage: nats-bench [-s server (%s)] [--tls] [-np NUM_PUBLISHERS] [-ns NUM_SUBSCRIBERS] [-n NUM_MSGS] [-ms MESSAGE_SIZE] [-csv csvfile] <subject>\n", nats.DefaultURL)
+	log.Printf("Usage: nats-bench [-s server (%s)] [--tls] [-np NUM_PUBLISHERS] [-ns NUM_SUBSCRIBERS] [-n NUM_MSGS] [-ms MESSAGE_SIZE] [-csv csvfile] <subject>\n", nats.DefaultURL)
+	for _, arg := range os.Args {
+		if strings.Contains(arg, "-help") {
```

Comment:
Can we not just have a normal bool flag for `--help`?

Reply:
> Can we not just have a normal bool flag for --help?

Yes, we can. But there is no good way to check either `-h` or `--help` command line argument is specified nor register callbacks for `-h` and `--help`.

Reply:
Ah ok. And docs say -h and --help or -help are all the same. So maybe we do go back to print usage followed by the defaults like you originally had. Apologies.